packages/ketcher-react/src/script/ui/dialog/toolbox/sgroup.module.less

```diff
@@ -36,8 +36,11 @@
       margin: 1px 0 0 3px;
     }
 
+    span {
+      margin-bottom: 4px;
+    }
+
     :global(.MuiOutlinedInput-root), input[type='text'] {
```

Comment:
It may be worth connecting HTML elements together using "&" and ">" so that styles are not overridden, just in case

Reply:
fixed